src/runtime/singlethreaded.rs

```diff
@@ -0,0 +1,83 @@
+use executor::current_thread::{self, CurrentThread};
+
+use tokio_reactor::{self, Reactor};
+use tokio_timer::timer::{self, Timer};
+use tokio_executor;
+
+use futures::Future;
+
+use std::io;
+
+/// Single-threaded runtime provides a way to start reactor
+/// and executor on the same thread
```

Comment:
Maybe add that this is not `Send` and can't be safely moved to other threads

Reply:
Done